src/workflow/ArcanistDiffWorkflow.php

```diff
@@ -1063,10 +1063,11 @@ protected function generateChanges() {
               "\n\n  $ %s\n",
               'svn propset svn:mime-type application/octet-stream <filename>'));
         } else {
+          $skip_confirmation = !$this->isTTY() && $this->getArgument('nointeractive');
```

Comment:
```suggestion
          $skip_confirmation = !$this->isTTY() || $this->getArgument('nointeractive');
```
Otherwise --no-interactive works in nonTTY world only, is this intended?